src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -1902,6 +1927,51 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
     Since `count(*)` does not name a specific attribute, the results will be formatted in the default ["humanize"](/docs/insights/new-relic-insights/managing-dashboards-data/data-formatter-setting-default-formats-numeric-values#format) format.
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="func-decode"
+    title={<InlineCode>decode(input, encoding)</InlineCode>}
+  >
+    Use the `decode()` to perform base64 conversions on strings and blobs. The input value (the first argument) will be decoded using the base64 standard specified by the encoding (the second argument).
```

Comment:
"Use the decode() function" or "Use decode()".  I think I'd hyphenate base-64 as well.

Reply:
(Also an evaluable)